requirements-dev.txt

```diff
@@ -1,14 +1,10 @@
 attrs >=17.4
-black
-bump2version >=1.0.1
-check-manifest >=0.36
-compas_invocations
-doc8
-flake8
-importlib_metadata <5.0
+black >=22.12.0
+bump-my-version
+compas_invocations2
+compas_notebook
```

Comment:
Is the dependency on `compas_notebook` intentional or leftover?

Reply:
leftover. will remove it...